map2csv/webtool_map2csv.py

```
@@ -0,0 +1,238 @@
+    if "homography" in st.session_state:
+        M = np.array(st.session_state["homography"])
+        if "clicked_points" not in st.session_state:
+            st.session_state["clicked_points"] = []
+        if "numbers" not in st.session_state:
+            st.session_state["numbers"] = []
+        img_final = img.copy()
+        draw_final = ImageDraw.Draw(img_final)
+        font = None
+        try:
+            font = ImageFont.truetype("DejaVuSans-Bold.ttf", 24)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)